packages/database/lib/modular/index.js

```diff
@@ -0,0 +1,124 @@
+import { firebase } from '..';
+
```

Comment:
We should probably port over `isEqual()` to modular as well to compare queries.

Reply:
Also no `off()` or `on()` or `once()` function for modular

Reply:
Explained above about `once`. As for `off` and `on`, you're expected to use the function returned from `on*` functions to unsubscribe. No functions exist for those in V9